examples/long_running_measurement/start.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+REM The discovery service uses this script to start the measurement service.
+REM You can customize this script for your Python setup. The -v option logs
+REM messages with level INFO and above.
+
+call python "%~dp0measurement.py" -v
```

Comment:
Looks like most / all of our other examples use `.venv\Scripts\python.exe measurement.py -v`. Is there a reason this is different?